content/en/cloudcraft/getting-started/group-by-presets.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Group By and Presets
+---
+
+Cloudcraft's **Group By** and **Presets** features offer powerful tools for creating focused, custom diagrams of your cloud infrastructure. These tools provide a structured approach to visualize cloud architecture, catering to specific use cases like infrastructure, network, and security requirements.
+
+By utilizing grouping and preset configurations, users can quickly generate focused diagrams, improving workflow efficiency and resource management. Whether you are troubleshooting, conducting security audits, or analyzing network performance, these features help you create focused, insightful diagrams quickly and easily.
+
+## Group By
+
+With **Group By**, Cloudcraft divides your diagram into distinct sections based on different group types. This feature provides a clear, organized view of your resources, even when they belong to multiple groups, which is particularly useful for visualizing complex cloud environments where resources may overlap across different categories.
+
+### AWS grouping options
+
+In AWS, you can group resources by:
+- Region
+- VPC
+- Security Group
+- Subnet
+- Network ACL
+
+### Azure grouping options
+
+In Azure, you can group resources by:
+- Resource Group
+- Region
+- VNet
+- Subnet
+
+## Presets
+
+**Presets** offer a convenient way to apply predefined sets of group-bys and filters, allowing you to quickly view your resources from different perspectives. This feature simplifies the process of applying groupings and filters to your diagrams, allowing you to focus on specific aspects of your architecture.
+
+**Cloudcraft provides three built-in presets:** Infrastructure, Network, and Security. These presets are designed to address different operational needs.
+
+{{< img src="cloudcraft/getting-started/group-by-presets/diagram-presets.png" alt="Cloudcraft interface showing preset options with the infrastructure diagram preset selected." responsive="true" style="width:100%;">}}
+
+To apply a preset to your diagram:
+
+1. Switch to the **Live** tab within Cloudcraft.
+2. Select the desired preset from the menu at the top of the diagram view.
+3. The diagram automatically updates to reflect the chosen preset.
+
+### Infrastructure diagram
+
+{{< img src="cloudcraft/getting-started/group-by-presets/infrastructure-diagram.png" alt="Infrastructure diagram showing servers, databases, security components, and the relationship between them." responsive="true" style="width:100%;">}}
+
+The infrastructure preset provides a broad overview, grouping resources by Region and VPC in AWS, and by Region and VNet in Azure. This preset is ideal for quickly generating architecture diagrams for troubleshooting or high-level review.
+
+- In AWS, it deactivates components like EBS, NAT Gateway, and Transit Gateway, among others.
```

Comment:
"deactivates" is an interesting way to frame this :P what about excludes? Could also be worth calling out something along the lines of "it by default excludes some components like EBS, NAT Gateway, and Transit Gateway, among others to give you an uncluttered diagram, showing you the most important parts of your architecture" to explain to users why some components aren't included

Reply:
Agree with @ruihanzhao - deactivates has a different meaning in the cloud world.

Reply:
Yeah, I could've used better wording on that one, haha. I'll work on fixing it.